src/main/java/mate/jdbc/Main.java

```diff
@@ -2,6 +2,6 @@
 
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        Application.run();
     }
```

Comment:
Why do we need additional class with method run() just to call it from main?
Isn't it easier to just place code in main method?

Reply:
I did it because I think If it's an application, it's better to create another class and put their code. 
As a result, I call run() in main.